src/app/app.module.ts

```diff
@@ -1,9 +1,10 @@
-import { NgModule } from '@angular/core';
-import { AppComponent } from './app.component';
-import { FacebookService } from './services/facebook.service';
-import { CoreModule } from './core/core.module';
-import { PagesModule } from './pages/pages.module';
-import {FaqService} from "./services/faq.service";
+import {NgModule} from '@angular/core';
+import {AppComponent} from './app.component';
+import {FacebookService} from './services/facebook.service';
+import {CoreModule} from './core/core.module';
+import {PagesModule} from './pages/pages.module';
+import {FaqService} from './services/faq.service';
+import {GuestsService} from './services/guests.service';
 
```

Comment:
Like, although I'd create a separate commit for that. Those changes have nothing to do with what's in the commit message. I like the approach: one change - one commit.